test/acceptance/query-page.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+describe('Query Page', function() {
+
+  beforeEach(function () {
+    browser.url('http://localhost:8001');
+  });
+
+  describe('the form', function () {
+
+    it('has a title', function () {
+      browser.getText('h1').should.equal('Applicants Details');
+    });
+
+    it('has the correct form elements', function () {
```

Comment:
Don't like it! Don't think we should use browser driven tests to check content on every page as brittle and makes for slow tests. Think for selenium/webdriver tests we should test behaviour of the form only. May need some content checking to make sure you're on the expected page, but don't like checking the text of every label.

If we do check content we could do it on the view layer - just render a view and check content - saves starting up a web server. But think we should only do content checking tests if we can extract the content into translations files or similar.